packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/TaskNavButtonsContainer.spec.js

```diff
@@ -1,77 +1,107 @@
-import { types } from 'mobx-state-tree'
 import React from 'react'
 import { shallow } from 'enzyme'
 import { expect } from 'chai'
+import { Provider } from 'mobx-state-tree'
+import { Factory } from 'rosie'
 import sinon from 'sinon'
 import TaskNavButtonsContainer from './TaskNavButtonsContainer'
-import ClassificationStore from '@store/ClassificationStore'
-import taskRegistry from '@plugins/tasks'
-import Step from '@store/Step'
-import { SubjectFactory, WorkflowFactory, ProjectFactory } from '@test/factories'
-
-const SingleChoiceTask = taskRegistry.get('single')
-const MultipleChoiceTask = taskRegistry.get('multiple')
 
-const project = ProjectFactory.build()
-const subject = SubjectFactory.build()
-const workflow = WorkflowFactory.build()
+import RootStore from '@store'
+import Step from '@store/WorkflowStepStore/Step'
+import { SubjectFactory, WorkflowFactory, ProjectFactory } from '@test/factories'
+import stubPanoptesJs from '@test/stubPanoptesJs'
 
 describe('TaskNavButtonsContainer', function () {
   function setupMocks () {
```

Comment:
It might be possible to import this function from a helper. It gets used in a couple of different test files.